scene/gui/color_picker.h

```diff
@@ -417,7 +418,7 @@ class ColorPickerButton : public Button {
 	bool is_editing_alpha() const;
 
 	ColorPicker *get_picker();
-	PopupPanel *get_popup();
+	PopupPanel *get_popup_panel();
```

Comment:
You can't rename this, it breaks compatibility, which indicates that this change isn't safe as the name change is based on that there's already a `get_popup` in `OptionButton`